src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -305,4 +305,23 @@ public Result saveManualAssessment(final Submission submission, final List<Feedb
         result.setAssessor(assessor);
         return result;
     }
+
+    /**
+     * Deletes the result of a submission.
+     *
+     * @param submission - the submission which the result belongs to
+     * @param result     - the result that should get deleted
```

Comment:
nit, maybe I would not use the "-" in oder to be consistent with the other doc strings in the controller; this would apply to all the other doc strings you introduced, so I guess you will keep it as it is :D

Reply:
Agree, also because in other places in this PR the documentation is done without the `-`